frontend/src/components/program/picasso/Picasso.vue

```diff
@@ -318,14 +318,18 @@ export default {
     },
     computedIntervalHeight () {
       return this.intervalHeight ??
-        this.$vuetify.breakpoint.xsOnly
+      this.$vuetify.breakpoint.xsOnly
         ? 1.3 * (this.$vuetify.breakpoint.height - 140) / this.intervalCount
         : 1.3 * Math.max((this.$vuetify.breakpoint.height - 204) / this.intervalCount, 32)
     }
   },
   mounted () {
-    this.period.camp().activities()._meta.load.then(() => { this.activitiesLoading = false })
-    this.period.camp().categories()._meta.load.then(() => { this.categoriesLoading = false })
+    this.period.camp().activities()._meta.load.then(() => {
+      this.activitiesLoading = false
+    })
+    this.period.camp().categories()._meta.load.then(() => {
+      this.categoriesLoading = false
+    })
 
     // scroll a bit down to hide the night hours
     const scroller = this.$el.querySelector('.v-calendar-daily__scroll-area')
```

Comment:
This auto-scrolling logic does not work anymore now on my machine. Fix it or remove it.

Reply:
Fixed in e38ac5bb